docs/source/project-management/work-management.rst

```diff
@@ -0,0 +1,83 @@
+.. _work-management:
+
+Work Management
+===============
+
+
+Work Management Overview
+------------------------
+
+The SDC uses Github issues to track work. These issues are organized into two types of tasks: high-level tasks and implementation tasks.
+
+- High-level tasks
+    - used to track the progress of each instrument/data level and infrastructure features (ie. Reprocessing high-level task)
+    - tasks should be fairly static throughout development and will only change if an instrument changes their data products or levels
+    - used to track the progress of implementation tasks
+    - managed in the high-level board
+    - labeled with the "Repo: Parent Issue" GitHub label
+- Implementation tasks
+    - define work that needs to be implemented
+    - each task should be small enough to be handled in a single PR
+    - organized as sub-tasks under high-level tasks
+    - should be added to high-level tasks as soon as they are identified
+    - should have work estimates tied to each task
+    - managed in the release boards
+
+The below image shows an example of a high-level task with sub-tasks for implementation tasks:
+
+.. image:: ../_static/high-level-task.png
+   :alt: High-level github task with sub-tasks
+
+
+
+Project Boards
+--------------
+
+The SDC utilizes two types of project boards for tracking work:
+
+- High-level board
+    - only tracks high-level tasks
+    - provides a quick view of the progress being made across instruments
+- Release board
+   - only tracks implementation tasks
+   - uses a two month rolling window for planning implementation tasks for the next two releases
+
+
+High-level Board
+~~~~~~~~~~~~~~~~
+
+The `high-level board <https://github.com/orgs/IMAP-Science-Operations-Center/projects/2/views/32>`_ contains a task for each instrument and data level. The tasks showing in this board should be fairly static throughout development and would only change if a instrument changes their data products or levels. This board provides a quick view of the progress being made across instruments.
+
+Release Board
+~~~~~~~~~~~~~
+
+The `release board <https://github.com/orgs/IMAP-Science-Operations-Center/projects/2/views/30>`_ allows for more detailed planning for the the current and upcoming release.
+Any tasks in the current release board that are not completed by the end of the release will be pushed to the next release.
```

Comment:
```suggestion
Any tasks in the current release board that are not completed by the end of the release will be pushed to the next release, but it acts as an estimate for work that will be completed that month.
```

Reply:
>but it acts as an estimate for work that will be completed that month.

Can you clarify what you mean by that? I'm not sure I'm understanding the intent

Reply:
Yeah, I'm thinking I just want to make it clear that the monthly release board is a work estimate for that month, and doesn't just include everything

Reply:
Ah gotcha sounds good. Thanks for the review!